core/src/components/picker-column/picker-column.tsx

```diff
@@ -167,12 +173,8 @@ export class PickerColumn implements ComponentInterface {
    */
   @Method()
   async setValue(value?: string | number) {
-    const { items } = this;
     this.value = value;
-    const findItem = items.find((item) => item.value === value && item.disabled !== true);
-    if (findItem) {
-      this.ionChange.emit(findItem);
-    }
```

Comment:
Not needed anymore. We could have this filter through the existing `ion-picker-column-option` elements, but none of our other grouped components do this (accordion, radio group, etc)